implementations/index.js

```diff
@@ -22,22 +21,29 @@ const getLocalManifest = fileName => {
     return require(path);
   } catch(e) {
     if(e?.code === 'MODULE_NOT_FOUND') {
-      return [];
+      return {};
     }
     throw e;
   }
 };
 
-// get all local endpoints dot file or otherwise
-manifests.local = [
-  '.localImplementationsConfig.cjs',
-  'localImplementationsConfig.cjs'
-].flatMap(path => getLocalManifest(path));
+const localConfig = getLocalManifest('localConfig.cjs');
+export const localSettings = (Object.keys(localConfig).length > 0) ?
+  // we have a localConfig object, so merge in local defaults
+  {...localConfig?.settings,
+    ...{enableInteropTests: false, testAllImplementers: false}} :
+  // otherwise, return the global defaults
+  // FIXME: ...consider renaming `localSettings` as it can hold global ones...
+  {enableInteropTests: true, testAllImplementers: true};
 
-// concat all the implementation manifests together
-const all = manifests.all = manifests.remote.concat(manifests.local);
+// get localConfig and look for an implementations property
+const local = localConfig?.implementations || [];
 
-// look for only in a local manifests
-const only = manifests.only = manifests.local.filter(i => i?.only === true);
+// concat all the implementation manifests together
+const all = remote.concat(local);
 
-export const implementerFiles = only.length ? only : all;
+// if local implementations are defined only return local implementations
+export const implementerFiles = local.length ?
```

Comment:
two tenary operators gets a bit hard to read maybe a helper functions here that returns an array to implementerFiles would help?

Reply:
Yeah...I figured we'd want to refactor this (eventually). Happy to do that as a cleanup step.